admin-dev/themes/default/template/controllers/countries/helpers/form/form.tpl

```diff
@@ -32,7 +32,7 @@
 					<textarea id="ordered_fields" name="address_layout" style="height:150px;">{$input.address_layout}</textarea>
 				</div>
 				<div class="col-lg-8">
-					{l s='Required fields for the address (click for more details):' d='Admin.International.Feature'}
+					{l s='Depending on your country, some elements can be required, you can click on "Use the default format" to get valid address format for this country.' d='Admin.International.Feature'}
```

Comment:
Hi @LouiseBonnard, as seen with @colinegin we are rewording this part, don't hesitate if you have ideas for a better formulation, the two informations that we want to pass are:

1/ Depending on the country the required fields are not always the same.
2/ There's a button just below to get a default address format for this country.

Reply:
Here is my suggestion, feel free to challenge it: _Some countries require different elements than others. Click on the button below to get the valid default address format for this country._ localized in the same domain. :-)

Reply:
👍 awesome, thank you !